cumulusci/tasks/metadeploy.py

```diff
@@ -0,0 +1,123 @@
+import json
+import requests
+
+from cumulusci.core.config import BaseConfig
+from cumulusci.core.exceptions import PlanNotFound
+from cumulusci.core.tasks import BaseTask
+from cumulusci.core.flowrunner import FlowCoordinator
+
+
+class Publish(BaseTask):
+    """Publishes an installation plan to MetaDeploy.
+    """
+
+    task_options = {
+        "plan": {
+            "description": "Name of the installation plan to publish",
+            "required": True,
+        },
+        "tag": {"description": "Name of git tag to publish"},
+    }
+
+    def _init_task(self):
+        metadeploy_service = self.project_config.keychain.get_service("metadeploy")
+        self.base_url = metadeploy_service.url
+        self.api = requests.Session()
```

Comment:
if we end up having more tasks that touch metadeploy, you can do something like BaseSalesforceTask, where `_init_api()` inits an api client at the api member. this works for now, but it is kinda nice to have the logic for constructing an api client chunked out